docs/chymyst01.md

```diff
@@ -278,7 +283,7 @@ val counter = new M[Int]("counter")
 
 ```
 
-Molecule names are very useful for debugging and logging.
+Descriptive names of molecules are very useful for visualizing the reactions, as well as for debugging and logging.
```

Comment:
I'd like to leave the short names in this example. The declarative nature of the dining philosophers solution is then more clearly seen.